test/distributed/org/apache/cassandra/distributed/test/MigrationCoordinatorTest.java

```diff
@@ -21,32 +21,43 @@
 import java.net.InetAddress;
 import java.util.UUID;
 
+import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
 import org.apache.cassandra.distributed.Cluster;
 import org.apache.cassandra.distributed.api.IInstanceConfig;
 import org.apache.cassandra.distributed.api.TokenSupplier;
 import org.apache.cassandra.distributed.shared.NetworkTopology;
+import org.apache.cassandra.distributed.shared.WithProperties;
 import org.apache.cassandra.schema.Schema;
 
-import static org.apache.cassandra.config.CassandraRelevantProperties.CONSISTENT_RANGE_MOVEMENT;
+import static org.apache.cassandra.config.CassandraRelevantProperties.BROADCAST_INTERVAL_MS;
 import static org.apache.cassandra.config.CassandraRelevantProperties.IGNORED_SCHEMA_CHECK_ENDPOINTS;
 import static org.apache.cassandra.config.CassandraRelevantProperties.IGNORED_SCHEMA_CHECK_VERSIONS;
 import static org.apache.cassandra.config.CassandraRelevantProperties.REPLACE_ADDRESS;
+import static org.apache.cassandra.config.CassandraRelevantProperties.RING_DELAY;
 import static org.apache.cassandra.distributed.api.Feature.GOSSIP;
 import static org.apache.cassandra.distributed.api.Feature.NETWORK;
 
 public class MigrationCoordinatorTest extends TestBaseImpl
 {
+    private static WithProperties withProperties;
```

Comment:
Just opinion - I do not know what is the history of this `withProperties` class  but it is more confusing when I read the test... Isn't it already a repetition to what we do in `CassandraRelevantProperties`? To me as a reader is more clear to just use` property_x.set`

Not saying you should change it

Reply:
oh, this is actually protected by checkstyle, it forced me to use it...

Reply:
Hmmm, it seems the `clearValue` had a checkstyle suppression added before this patch. 
:D I was not the only one who did not like it I guess hehe

Reply:
I can imagine more user-friendly api for `WithProperties`... but it would be good to use it or implement something in common pre/post test hooks, separate topic

Reply:
Agreed, not something to address here